packages/components/src/button/docs/Button.stories.mdx

```diff
@@ -34,6 +34,28 @@ import { Text } from "@components/typography";
     githubPath="/packages/components/src/button/src"
 />
 
+## Guidelines
+
+### When to use
+
+- To trigger an action such as submitting a form, exporting data.
+
+### Content
+
+Buttons should:
+
+- Use strong, actionable verbs.
+- Be positionned in consistent locations.
+
+### Accessibility
+
+- Use the disabled prop to set the disabled state of the button.
+
+### Button vs Link vs Switch
+
+- When you need to link to another page, use a [link](?path=/docs/link--default-story) component.
+- To activate or deactivate a feature, use the [switch](?path=/docs/switch--default-story) instead of a single [toggle button](?path=/docs/button--default-story#toggle-button).
```

Comment:
I would split those in 2 sections since a basic "button" is not related at all with a switch.

I would do

- Button vs Link
- ToggleButton vs Switch